.travis.yml

```diff
@@ -16,4 +15,4 @@ addons:
     - libclang-common-3.4-dev
     - clang-3.4
 
-script: util/travis
+script: util/travis_${TRAVIS_OS_NAME}.sh
```

Comment:
Since these scripts are not Boune shell, it would be better to drop `.sh` (no point in adding extra info that is not accurate).